media/server/main/source/TextTrackAccessor.cpp

```diff
@@ -67,7 +67,7 @@ std::optional<uint32_t> TextTrackAccessor::openSession(const std::string &displa
     uint32_t result = m_textTrackWrapper->openSession(displayName, sessionId);
     if (m_thunderWrapper->isSuccessful(result))
     {
-        RIALTO_SERVER_LOG_INFO("TextTrack session %u created with display '%s'", sessionId, displayName.c_str());
```

Comment:
Why MIL here? We have MIL prints for "global" play/pause/create/close session etc.